core/templates/components/question-directives/modal-templates/remove-question-skill-link-modal.component.html

```diff
@@ -40,8 +40,7 @@ <h3 *ngIf="!topicsAssignmentsAreFetched">
   </div>
 </div>
 <div class="modal-footer">
-  <button class="btn btn-secondary" (click)="cancel()">Cancel</button>
-  <button class="btn btn-danger"
+  <button class="btn btn-danger e2e-test-remove-question-confirmation-button"
```

Comment:
You are deleting a button instead of just adding the class to the button. Is that intended? What happened to the button that you deleted?

Reply:
yes, not intended.. updated. 
Thanks for noticing.